### DISCUSSION TITLE
Videos don't load

### DISCUSSION THREAD

**Discussion by lockinmarv** [go-rod/rod#1104] [Q&A] [answered]
Hey! I'm just starting to use Rod but can't figure out how to show videos on a website.

They are always at 0:00 and with gray background like this:
![CleanShot 2024-08-25 at 08 42 38@2x](https://github.com/user-attachments/assets/e21fc8ad-d661-46a9-b9b1-50f43889c68d)

I can't see any errors in the console either. When I open it directly (non headless), it can't get loaded.

Do I need to configure something to make it work?

**Comment by ysmood** [comment, marked as answer]
It might be some license issue, chromium doesn't support all the video types, use chrome might work, it supports more paid licenses.

https://go-rod.github.io/#/custom-launch?id=connect-to-an-running-browser

**Reply by lockinmarv** [reply]
Thanks, this works! 🤝
